guides/source/active_record_encryption.md

```diff
@@ -410,19 +410,21 @@ config.active_record.encryption.extend_queries = true
+| `excluded_from_filter_parameters`              | You can configure a list of params that won't be filtered out when `add_to_filter_parameters` is true. Default: [].                                                                                                                                                                                                                                                            |
+| `validate_column_size`                         | Adds a validation based on the column size. This is recommended to prevent storing huge values using highly compressible payloads. Default: true.                                                                                                                                                                                                                              |
+| `primary_key`                                  | The key or lists of keys that is used to derive root data-encryption keys. They way they are used depends on the key provider configured. It's preferred to configure it via a credential `active_record_encryption.primary_key`.                                                                                                                                              |
+| `deterministic_key`                            | The key or list of keys used for deterministic encryption. It's preferred to configure it via a credential `active_record_encryption.deterministic_key`.                                                                                                                                                                                                                       |
+| `key_derivation_salt`                          | The salt used when deriving keys. It's preferred to configure it via a credential `active_record_encryption.key_derivation_salt`.                                                                                                                                                                                                                                              |
+| `forced_encoding_for_deterministic_encryption` | The default encoding for attributes encrypted deterministically. You can disable forced encoding by setting this option to `nil`. It's `Encoding::UTF_8` by default.                                                                                                                                                                                                           |
+| `hash_digest_class`                            | The digest algorithm used to derive keys. `OpenSSL::Digest::SHA256` by default.                                                                                                                                                                                                                                                                                                |
```

Comment:
Big diff because I formatted the markdown table. It's also adding the last line (`hash_digest_class`)